far/changelog_eng

```diff
@@ -3,6 +3,10 @@ This file is a translation of the main russian changelog and is provided by volu
 It might not always be as up to date as the main changelog.
 =======================================================================================
 
+MZK 01.04.2018 21:23:21 -0700 - build 5178
+
+1. Починил переполнение буфера в SearchString + рефакторинг.
```

Comment:
Looks like changelog entries are mixed up.

Reply:
Thank you. Fixed.